codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesresponse.java

```diff
@@ -1254,9 +1251,15 @@ default Builder polymorphicTypeWithoutSubTypes(Consumer<SubTypeOne.Builder> poly
          * @see EnumType
          */
         Builder enumType(EnumType enumType);
+
+        @Override
+        Builder responseMetadata(AwsResponseMetadata awsResponseMetadata);
```

Comment:
I think this has been removed?

Reply:
Do you mean for now? Or we'll never expose response metadata?

Reply:
For now. It's scheduled as a post-GA task for now.

Reply:
Sounds good; i'll remove it